XaiValidatePublicApiDefinition.yml

```diff
@@ -4172,12 +3944,20 @@ components:
                         HasHecsDebt:
                           type: boolean
                           description: Determines whether the applicant has a HECS debt to pay.
+                        MaritalStatus:
```

Comment:
@rjshanahan could you explain why this is necessary?

Reply:
`MaritalStatus` is used to drive the HEM lookup value (along with income and dependants)